bdk_core/src/coin_select.rs

```diff
@@ -1,54 +1,63 @@
 use crate::{collections::BTreeSet, Vec};
 use bitcoin::{LockTime, Transaction, TxOut};
 
-pub const TXIN_BASE_WEIGHT: u32 = (32 + 4 + 4) * 4;
+pub const TXIN_FIXED_WEIGHT: u32 = (32 + 4 + 4) * 4;
 
 #[derive(Debug, Clone)]
 pub struct CoinSelector<'a> {
-    candidates: Vec<WeightedCandidate>,
+    candidates: Vec<InputCandidate>,
     selected: BTreeSet<usize>,
     opts: &'a CoinSelectorOpt,
 }
 
 #[derive(Debug, Clone, Copy)]
-pub struct WeightedCandidate {
+pub struct InputCandidate {
     pub value: u64,
-    /// Typically set as [`TXIN_BASE_WEIGHT`] for a single Bitcoin input.
-    pub base_weight: u32,
+    /// Weight of the "fixed" fields of a `txin` (`prevout` + `nSequence`).
+    /// Typically set as [`TXIN_FIXED_WEIGHT`] for a single Bitcoin input.
+    pub fixed_weight: u32,
+    /// Weight of the "satisfaction" fields of a `txin` (`scriptSig` + `scriptWitness`).
     pub satisfaction_weight: u32,
```

Comment:
can we not just merge `fixed_weight` and `satisfaction_weight` and call it `inclusion_weight`?

Reply:
I think we can

Reply:
I just called it `weight`, hope that is okay?